File: LV0/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        int answer = 0;
        if(eq.equals("=")){
            return (ineq.equals(">") && n >= m) || (ineq.equals("<") && m >= n) ? 1 : 0;
        } else if (eq.equals("!")){
            return (ineq.equals("<") && n <= m) || (ineq.equals(">") && m <= n) ? 1 : 0;
        }
        return answer;
    }
}

File: LV0/조건 문자열.go
func solution(ineq string, eq string, n int, m int) int {
    if eq == "!" {
        if ineq == "<" {
            return int(n < m)
        } else {
            return int(n > m)
        }
    } else {
        if ineq == "<" {
            return int(n <= m)
        } else {
            return int(n >= m)
        }
    }
}